monkey/infection_monkey/monkey.py

```diff
@@ -494,6 +494,7 @@ def _stop_relay(self):
     def _publish_agent_shutdown_event(self):
         agent_shutdown_event = AgentShutdownEvent(source=self._agent_id, timestamp=time.time())
         self._agent_event_queue.publish(agent_shutdown_event)
+        self._island_api_client.send_events([agent_shutdown_event])
```

Comment:
This is island side, so these changes shouldn't be here?

Reply:
I updated the other branch which caused it to show up as a new change on this one. Rebased now.